catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/PaimonCatalogPropertiesMetadata.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.lakehouse.paimon;
+
+import static com.datastrato.gravitino.connector.PropertyEntry.enumImmutablePropertyEntry;
+import static com.datastrato.gravitino.connector.PropertyEntry.stringRequiredPropertyEntry;
+
+import com.datastrato.gravitino.connector.BaseCatalogPropertiesMetadata;
+import com.datastrato.gravitino.connector.BasePropertiesMetadata;
+import com.datastrato.gravitino.connector.PropertyEntry;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Implementation of {@link BasePropertiesMetadata} that represents Paimon catalog properties
+ * metadata.
+ */
+public class PaimonCatalogPropertiesMetadata extends BaseCatalogPropertiesMetadata {
+
+  public static final String CATALOG_BACKEND = "catalog-backend";
+  public static final String WAREHOUSE = "warehouse";
+
+  private static final Map<String, PropertyEntry<?>> PAIMON_CATALOG_PROPERTY_ENTRIES;
+
+  static {
+    List<PropertyEntry<?>> propertyEntries =
+        ImmutableList.of(
+            enumImmutablePropertyEntry(
+                CATALOG_BACKEND,
+                "Paimon catalog type choose properties",
```

Comment:
change description to `Paimon catalog backend type`?  the description in `IcebergCatalogPropertiesMetadata` is not proper.